pom.xml

```diff
@@ -67,6 +67,7 @@
         <module>streampark-flink</module>
         <module>streampark-spark</module>
         <module>streampark-console</module>
+        <module>streampark-flink/streampark-flink-shims/streampark-flink-shims_flink-1.20</module>
```

Comment:
According to the current architecture design, there is no need to define the flink-1.20 module here. Could you confirm this?